File: src/main/java/com/example/ssauc/user/cash/dto/ChargeRequestDto.java
package com.example.ssauc.user.cash.dto;

import lombok.Data;

@Data
public class ChargeRequestDto {
    // 결제 완료 시 클라이언트에서 서버로 전달되는 정보
    private String paymentId;
    private Long amount;
}

File: src/main/java/com/example/ssauc/user/cash/dto/ChargeRequestDto.kt
package com.example.ssauc.user.cash.dto

import lombok.Data

@Data
class ChargeRequestDto {
    // 결제 완료 시 클라이언트에서 서버로 전달되는 정보
    private val paymentId: String? = null
    private val amount: Long? = null
}